docs/tutorials/AddRemark.md

```diff
@@ -308,9 +308,9 @@ Just add [this one line of code!](https://github.com/se-edu/addressbook-level3/c
 **`PersonCard.java`:**
 
 ``` java
-public PersonCard(Person person, int displayedIndex) {
+public PersonCard(Person task, int displayedIndex) {
```

Comment:
This is still not changed

Reply:
These are in addremark.md not actual code, not sure if it needs to be changed

Reply:
ok undid changes in these documents, please review again